src/components/ExtensibilitySpectrum/index.jsx

```diff
@@ -0,0 +1,177 @@
+import React from "react";
+import styles from "./index.module.css";
+import classNames from "classnames";
+
+const appConfigSvg = (
```

Comment:
Can we turn these into functions that return the JSX?

`function AppConfigIcon() { ... }` and use like `<AppConfigIcon />` instead.

Reply:
Done.

Reply:
I notice we're using a mix of `function` and arrow function constants. Can we use `function` for all?

Reply:
Good call. Changed all to function.